Portal/src/Datahub.Infrastructure/ConfigureServices.cs

```diff
@@ -55,7 +55,7 @@ public static class ConfigureServices
         {
             x.UsingAzureServiceBus((context, cfg) =>
             {
-                cfg.Host(configuration["DatahubServiceBus:ConnectionString"]);
```

Comment:
why are we doing this change? I just went through poc configuration that this is the environment variable that is configured

Reply:
Fixed (my local config was a bit different)